object/virtual_machine.go

```diff
@@ -689,3 +689,24 @@ func (v VirtualMachine) Unregister(ctx context.Context) error {
 	_, err := methods.UnregisterVM(ctx, v.Client(), &req)
 	return err
 }
+
+// QueryEnvironmentBrowser is a helper to get the environmentBrowser property.
+func (v VirtualMachine) QueryEnvironmentBrowser(ctx context.Context) (*types.ConfigTarget, error) {
```

Comment:
Let's name this method 'QueryConfigTarget', since 'EnvironmentBrowser' is just a parameter.